include/cblox/core/tsdf_submap_collection.h

```diff
@@ -57,6 +61,13 @@ class TsdfSubmapCollection {
     return tsdf_sub_maps_.back()->getPose();
   }
 
+  // Gets the ID of the patch on the tip of the collection
+  const SubmapID getActiveSubMapID() const {
+    if (!tsdf_sub_maps_.empty())  // First check if the collection is not empty
```

Comment:
This check is a good idea. I hadn't thought about that. However,
a) As far as I can tell 0 is a valid SubmapID so returning 0 doesn't totally alert the user to the fact that something has gone wrong.
b) All of our other functions don't check for empty collection. I would actually just remove this check and put the responsibility on the caller to check for this case.

Reply:
Ok, I'll change it back.